products/forms.py

```diff
@@ -0,0 +1,60 @@
+from django import forms
+from .models import Brand
+from .messages import BrandFormErrorMessages
+
+
+class AddBrandForm(forms.ModelForm):
+    """" form to update brand details """
+
+    name = forms.CharField(
+        max_length=40,
+        required=True,
+        error_messages={'invalid': BrandFormErrorMessages.BRAND_NAME},
+        widget=forms.TextInput(
+            attrs={
+                'placeholder': 'Brand Name',
+                'class': 'form-control'
+            }
+        )
+
+    )
+
+    image = forms.ImageField(
```

Comment:
@rinkishivani why do we need max_length in ImageField? kindly give the specific field wise validation messages as here you have used same message for name and image.

cc: @lalitvasoya

Reply:
removed unnecessary validation.